content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fariness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+### Priority level
+
+Each apiserver has an aggregate concurrency limit and divides it ---
+without examining the request load --- among several priority levels.
+The priority levels operate independently of each other.  That is, the
+apiserver classifies a given request to exactly one priority level
+and, for a given priority level, handles at most the corresponding
+number of requests at any given time.  One special priority level ---
+used for meta-requests and other top-priority traffic --- imposes no
+limit.  This priority level and the requests mapped to it are
+described as "exempt", because of the lack of concurrency limitation.
+
+For a non-exempt request, if it is not allowed to be handled
+immediately then the apiserver will either reject the request or hold
+it in a queue for a while.  Each non-exempt priority level has a
+number of FIFO queues.  Whenever that priority level's number of
+requests currently being handled drops below the concurrency limit,
+the apiserver draws requests to begin handling from these queues using
+a technique that is an appropriate generalization of
+[fair queuing](https://en.wikipedia.org/wiki/Fair_queuing).
+
+### Flow schema
+
+To determine how to control a request, an apiserver first classifies
+that request to a _flow_ according to a configured collection of _flow
+schemas_.  The flow schemas are considered in order, according to each
+flow schema's _matching precedence_ (which is a number).  A flow
+schema has rules that describe which requests match that schema.
+Matching takes into account (a) the requested operation --- as
+identified by the HTTP request verb and URI --- and (b) if the request
+arrived through a secured port then the authenticated user attributes
+otherwise just the UserAgent.  Among the schemas that match the
+request, one with the logically highest matching precedence wins.
+
+A flow schema also has a declaration of how to complete the _flow
+identifier_ for the request's flow.  A flow identifier is a pair of
```

Comment:
Could you provide a code example (could be partial code sample) of a flow schema?

Reply:
Would pointing into the KEP be appropriate?

Reply:
I would be in favor of a real example.
Which KEP are you referring to?

Reply:
Opinions, based on what I've observed:

I think it's most common to put an example resource (eg in YAML) either in the Concept page or in a Task page. If the YAML is usable unmodified, and you feel like having a separate Task could make the Concept page more readable, by shortening it, then I favor that approach.

If the YAML is just an example that the reader can't use as-is, the Concept page is probably the best home for it (and it's OK to repeat that YAML in the Reference docs if appropriate).